test/integration/features/generic-driver.feature

```diff
@@ -0,0 +1,61 @@
+@generic-driver
+Feature: With generic driver Minishift can provision remote unprovisioned VM 
+
+  Scenario: User creates an unprovisioned VM
+    Given setting up environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" with value "y" succeeds
+     When executing "minishift profile set remote" succeeds
+      And executing "minishift start --no-provision" succeeds
```

Comment:
so useful ;-)

Reply:
@gbraad I think  it is time we make ` --no-provision` non experimental feature.